src/server-extension/resolvers/ChannelsResolver/index.ts

```diff
@@ -223,6 +231,8 @@ export class ChannelsResolver {
         timestamp: new Date(),
       })
 
+      recommendationServiceManager.scheduleItemBookmark(`${channel.id}-channel`, user.id)
```

Comment:
If upsert and deletion doesn't require following `${string}-video` template and scheduling user interactions, change the method param type to reflect it